### ISSUE TITLE
While dragging tabs around, it does not automatically scroll.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#45]
Originally reported on Google Code with ID 45

```
What steps will reproduce the problem?
1. Have a window that is long enough to produce a vertical scrollbar.
2. Start dragging another tab into this window, but don't release it.
3. While still dragging the tab, move it near the top or bottom of the window (with
scrollbars).

What is the expected output? What do you see instead?
The window should scroll up/down. Instead, it just sits there, and the only way to
scroll is to first cancel the drag, scroll manually, then start the drag again.

Please provide any additional information below.
This is a problem in Chrome. Looks like we'll need to wait for WebKit to fix this,
and then have the fix merged into Chrome.
```

Reported by `pcxunlimited` on 2010-09-16 12:12:05
- **Blocked on**: #37320, #38592

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-18 17:27:21
- **Labels added**: Usability

**Comment by Pauan** [comment]
```
I think this is annoying enough to count as higher priority.
```

Reported by `pcxunlimited` on 2010-09-28 05:42:09
- **Labels added**: Priority-High
- **Labels removed**: Priority-Medium

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:04:22
- **Labels added**: Type-External
- **Labels removed**: Type-Bug

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:08:06
- **Labels added**: Type-Internal
- **Labels removed**: Type-External

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:14:51
- **Labels removed**: Usability

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:17:41
- **Labels removed**: OS-All

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:39:18
- **Labels removed**: Priority-High

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:47:45
- **Status changed**: `External`
